### ISSUE TITLE
Local test of fedora upgrade process

### ISSUE THREAD

**Issue by bess** [curationexperts/alexandria#35]
**ACCEPTANCE CRITERIA**
- [x] Create a local installation of Fedora 4.6.0
- [x] Create some number of objects in it
- [x] Create a local installation of Fedora 4.7.5
- [x] Using command line tools, migrate objects from 4.6.0 - 4.7.5 and document the process

**Comment by shanitajohn** [comment]
Time-boxed at 5 points.

**Comment by val99erie** [comment]
If we use [the fcrepo-import-export tool](https://github.com/fcrepo4-labs/fcrepo-import-export) to export records from fedora 4.6.0 and import them into a new fedora 4.7.5, it looks like we'll need to use the `--legacyMode` option.

When using `--legacyMode`, there are some fields where the values in the new fedora won't match the values in the old fedora:

- fedora:lastModified
- fedora:lastModifiedBy
- fedora:created
- fedora:createdBy

I ran a test with a `Collection` record.  I used the CSV importer to create a Collection record in fedora 4.6.0, and then used the `fcrepo-import-export` tool to export it and import it into a fedora 4.7.5 instance.

The values for `createdBy` and `lastModifiedBy` actually did match between the old and new fedora.  The value was `bypassAdmin`, which I supposed is meant to indicate that the record was created by an importer rather than a user.

But the values for `lastModified` and `created` were different.  In the new fedora, they were set to the creation date of the record (the time when I ran the `fcrepo-import-export` tool).

My instinct is that it shouldn't matter if fedora's create and modify timestamps don't match between the old and new fedora, but I wanted to check with UCSB team to be sure.

@chrissyrissmeyer and @dunn - Do you think it would be a problem if the create/modify times don't match in the new fedora?

**Comment by dunn** [comment]
I can't see why that would matter, no.

**Comment by chrissyrissmeyer** [comment]
I can't see why it would matter either (especially given the number of times we have had to re-ingest/migrate)

**Comment by val99erie** [comment]
- I imported a collection with 2 cylinder records with attached audio files into fedora 4.6.0
- I used `fcrepo-import-export` tool to export the records from fedora 4.6.0
- I used `fcrepo-import-export` tool to import the records into fedora 4.7.5

Aside from the issue described in #49, everything looks good.  The records look correct in the UI, and I was able to play the audio in the browser.

The `fcrepo-import-export` tool has pretty good documentation.

I wrote some additional notes here:
[Fedora upgrade notes](https://gist.github.com/val99erie/b99645d070877671d89fb3d9002b1af3)

I'm going to close this ticket because it was just meant to be a quick test in a development environment.
